io.github.mfat.systemdpilot/io.github.mfat.systemdpilot.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.mfat.systemdpilot
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: systemd-pilot
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --system-talk-name=org.freedesktop.systemd1
+  - --system-talk-name=org.freedesktop.login1
+  - --socket=session-bus
+  - --system-talk-name=org.freedesktop.PolicyKit1
+  - --socket=system-bus
+  - --filesystem=xdg-run/gvfsd
+  - --filesystem=xdg-config/gtk-4.0:ro
+  - --filesystem=xdg-config/gtk-3.0:ro
+modules:
+  - name: systemd-pilot
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/python3/dist-packages/systemd_pilot
+      - cp src/main.py /app/lib/python3/dist-packages/systemd_pilot/
+      - mkdir -p /app/bin
+      - echo '#!/usr/bin/env python3' > /app/bin/systemd-pilot
+      - echo 'import sys' >> /app/bin/systemd-pilot
+      - echo 'sys.path.append("/app/lib/python3/dist-packages")' >> /app/bin/systemd-pilot
+      - echo 'from systemd_pilot.main import app' >> /app/bin/systemd-pilot
+      - echo 'app.run(None)' >> /app/bin/systemd-pilot
+      - chmod +x /app/bin/systemd-pilot
+      - install -Dm644 data/org.mfat.systemdpilot.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 systemd-pilot.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+    sources:
+      - type: git
+        url: https://github.com/mfat/systemd-pilot.git
+        branch: main
```

Comment:
Launching these from inside the Flatpak won't work https://github.com/mfat/systemd-pilot/blob/3fa255515cc8acecb01472d526bdd3a9f75d3ea9/src/main.py#L668-L687

https://github.com/mfat/systemd-pilot/blob/3fa255515cc8acecb01472d526bdd3a9f75d3ea9/src/main.py#L617

https://github.com/mfat/systemd-pilot/blob/3fa255515cc8acecb01472d526bdd3a9f75d3ea9/src/main.py#L212

The fundamental idea of the application seems incompatible with the sandboxing, you cannot easily edit systemd services from inside the sandbox without flatpak-spawn permission.

I don't think this app is a god fit for Flatpak.

Reply:
I did use flatpak-spawn and the app works fine from my locally built bundle.

Reply:
It works but having `flatpak-spawn` is equivalent to running it unsandboxed outside on host.